interfaces/hotplug/deviceinfo.go

```diff
@@ -89,3 +90,43 @@ func (h *HotplugDeviceInfo) Attribute(name string) (string, bool) {
 	val, ok := h.Data[name]
 	return val, ok
 }
+
+func (h *HotplugDeviceInfo) firstAttrValueOf(tryAttrs ...string) string {
+	for _, attr := range tryAttrs {
+		if val, ok := h.Attribute(attr); ok && val != "" {
+			return val
+		}
+	}
+	return ""
+}
+
+func (h *HotplugDeviceInfo) String() string {
+	var str []string
+
```

Comment:
Without as much background as you I can only speculate on the rationale about this. Could you please add a comment about why we prefer `devname` to `devpath` please?

Reply:
DEVNAMEis is the path under /dev, e.g. /dev/ttyUSB0. DEVPATH is a sysfs path; I think the former is more interesting (but no hard opinion on this, happy to change it if we find otherwise).

Reply:
I don't have an opinion. I just wanted you to write down yours in the code as a comment.